vyper/signatures/event_signature.py

```diff
@@ -69,13 +69,12 @@ def from_declaration(cls, code, global_ctx):
                     raise EventDeclarationException("Indexed arguments are limited to 32 bytes")
                 if topics_count > 4:
                     raise EventDeclarationException(
-                        f"Maximum of 3 topics {topics_count - 1} given",
-                        arg,
+                        f"Maximum of 3 topics {topics_count - 1} given", arg_item,
```

Comment:
Isn't this being triggered in another place in this PR?

Reply:
Ahh yeah.  I should be raising internal exceptions here.  will modify.

Reply:
I mean, you're doing the same check twice, why is that?

Reply:
Temporary sanity check while type checking pass is new and old code hasn't been refactored.